docs/openapi3.yaml

```diff
@@ -1890,6 +1893,185 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/upload_file_response'
+  /v1/chaos-experiments:
+    parameters:
+      - $ref: '#/components/parameters/context_id'
+    post:
+      operationId: create-chaos-experiment
+      tags:
+        - Chaos Experiments
+      summary: Create chaos file
+      description: >-
+        Create a chaos experiment.
+      responses:
+        '201':
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/chaosExperiment'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '422':
+          description: Unprocessable entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error_response'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/chaosExperiment'
+        description: The chaos experiment to create
+        required: true
+    get:
+      operationId: retrieve-all-choos-experiments
+      tags:
+        - Chaos Experiments
+      summary: Retrieve all chaos experiments
+      description: Retrieve all chaos experiments
+      parameters:
+        - in: query
+          name: from
+          description: From which result to start retrieve data - pagination
+          required: false
+          schema:
+            type: integer
+            minimum: 0
+            default: 0
+        - in: query
+          name: limit
+          description: Max results to return from this query ( Limit to 100 )
+          required: false
+          schema:
+            type: integer
+            maximum: 200
+            default: 100
+        - in: query
+          name: template
+          description: filter to get templates only
+          required: false
+          schema:
+            type: boolean
+            example: true
+        - in: query
+          name: exclude
```

Comment:
why do we need this?

Reply:
I'd just a common option in the crud sometimes not to show everything